protect/create.sh

```diff
@@ -0,0 +1,112 @@
+sudo rm -r Toil-toil-bwa
+rm runtime-container.DONE
+
+#s3am download --sse-key-file /home/alex/.aws/master.key --sse-key-is-master s3://cgl-driver-projects-encrypted/wcdt/oicr_issue42_input/RNA150410JA_10_1-1-15_M12_EV_05uM_JQ1.tar ./rna1.bam;
+#s3am download --sse-key-file /home/alex/.aws/master.key --sse-key-is-master s3://cgl-driver-projects-encrypted/wcdt/oicr_issue42_input/RNA150410JA_1_10-14-14_abl_0uM_JQ1.tar ./rna2.bam;
+#s3am download --sse-key-file /home/alex/.aws/master.key --sse-key-is-master s3://cgl-driver-projects-encrypted/wcdt/oicr_issue42_input/RNA150410JA_2_10-14-14_abl_05uM_JQ1.tar ./rna3.bam;
+#s3am download --sse-key-file /home/alex/.aws/master.key --sse-key-is-master s3://cgl-driver-projects-encrypted/wcdt/oicr_issue42_input/RNA150410JA_19_1-1-15_22RV1_0uM_JQ1.tar ./rna4.bam
+
+# Get the hash from the build of the newest docker container version
+variable=`make 2>/dev/null | grep "Successfully built" | sed -n '6'p | awk '{print $3}'`
+
+clear
+
+docker run \
+    -v $(pwd):$(pwd) \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    quay.io/ucsc_cgl/rnaseq-cgl-pipeline \
+    --star /inputs/starIndex_hg38_no_alt.tar.gz \
+    --rsem /inputs/rsem_ref_hg38_no_alt.tar.gz \
+    --kallisto /inputs/kallisto_hg38.idx \
+    --samples $(pwd)/rna1.tar $(pwd)/rna2.tar $(pwd)/rna3.tar $(pwd)/rna4.tar
+
+exit
+
+# Test data run
+nohup docker run -v /var/run/docker.sock:/var/run/docker.sock \
+    -v $(pwd):$(pwd) \
+    222d54951a57 \
+    --manifest \
+    --samples \
+    real_uuid \
+    $(pwd)/r1.fq \
+    $(pwd)/r2.fq \
+    --ref $(pwd)/ref \
+    --amb $(pwd)/amb \
+    --ann $(pwd)/ann \
+    --bwt $(pwd)/bwt \
+    --fai $(pwd)/fai \
+    --pac $(pwd)/pac \
+    --sa $(pwd)/sa \
+    --output-dir $(pwd)/ \
+    --sort \
+    &>final.txt & cat final.txt
+
+exit 
+
+patients:
+    TEST:
+        tumor_dna_fastq_1 : S3://cgl-protect-data/ci_references/Tum_1.fq.gz
```

Comment:
@natanlao looks like there is test data here, when you get a chance can you try running it on a c3.8xlarge EC2 instance?